src/algebra/algebra/operations.lean

```diff
@@ -287,6 +287,30 @@ lemma le_div_iff {I J K : submodule R A} : I ≤ J / K ↔ ∀ (x ∈ I) (z ∈
 lemma le_div_iff_mul_le {I J K : submodule R A} : I ≤ J / K ↔ I * K ≤ J :=
 by rw [le_div_iff, mul_le]
 
+lemma one_le_div {I : submodule R A} :
+  I ≤ (1 : submodule R A) ↔ 1 ≤ 1 / I :=
+begin
+  split, all_goals {intro hI},
+  {rw [le_div_iff_mul_le, one_mul], assumption },
```

Comment:
Seems like you do this `rw` in both subproofs, so you could do it before the `split` instead.
(Also, note that `rwa [..]` is short for `rw [..], assumption`.)

Reply:
Thanks for the `rwa[...]`; on the other hand, the `rw` is not `at` the same point, I am not sure how to perform it before `split`.